index.html

```diff
@@ -53,7 +53,7 @@
   </div>
   <div class="item-menu"><h3 onclick="openMain()"href="#" class="openMenu"><i class="fa fa-bars" aria-hidden="true"></i></h3></div>
       <div class="hello"><p>"Hello! Sing for peace"</p></div>
-      <p id="jedan">EXIT Music Festival Novi Sad 2022</p>
+      <h1 id="jedan">EXIT Music Festival Novi Sad 2022</h1>
       <p id="h1"class="con">Bringing together through culture and providing a space for
         celebration where people from all over the world can share their
         cultures.</p>
```

Comment:
- [ **OPTIONAL** ] As this detail was not pointed out by the previous reviewer, I think it would be better if you reduce the spacing between the elements from the **hero section** and also remember to match `font-size` used on each text element to the one from the **[Behance Template](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015)** 
Below is how looks your design:
![Screenshot from 2023-01-14 14-36-01](https://user-images.githubusercontent.com/83042742/212472007-038a7d54-2a47-4341-b482-1762f4a750cd.png)